components/dashboard/src/components/SelectWorkspaceClassComponent.tsx

```diff
@@ -42,11 +44,34 @@ export default function SelectWorkspaceClassComponent({
     );
 
     const getElements = useCallback((): ComboboxElement[] => {
-        return (workspaceClasses || [])?.map((c) => ({
+        const elements = (workspaceClasses || [])?.map((c) => ({
             id: c.id,
             element: <WorkspaceClassDropDownElement wsClass={c} />,
             isSelectable: true,
+            customParentClass: "",
         }));
+        if (isGitpodIo()) {
+            elements.push({
+                id: "learn-more",
+                element: (
+                    <div className="py-2 text-pk-content-secondary ">
+                        <span className="font-bold text-sm ">Need more classes? </span>
+                        <a
+                            className="text-sm inline-flex"
+                            href="https://www.gitpod.io/docs/enterprise"
+                            target="_blank"
+                            rel="noreferrer"
+                        >
+                            Learn about Gitpod Enterprise
+                            <FullArrow className="ml-2" />
+                        </a>
+                    </div>
+                ),
+                customParentClass: "justify-end",
```

Comment:
Do you think we can make this class a part of element so that we don't need to change podkit?

Reply:
I've tried that initially, but the parent element (which is coming from podkit) is controlling the alignment.

Reply:
We could try to wrap with a dev, and make it `w-full flex justify-end`

Reply:
This got resolved in [7ae9c46](https://github.com/gitpod-io/gitpod/pull/19775/commits/7ae9c46c86460610305f691362f39d122d842746)